### ISSUE TITLE
[FEATURE REQUEST]: Add modifier to popup of DisplayResult

### ISSUE THREAD

**Issue by wcroteau** [3d-dice/FUI#7]
At the moment when your roll includes a modifier the DisplayResult component properly displays the modified total however there's no indication in the calculation that there was a modifier at play, it simply shows the dice rolled = total.

This is particularly noticeable when you roll a single die with modifier.  For example Roll('1d20', { modifier: 3 } results in "16 = 19" (when the die roll is 16 which is mathematically nonsense at best and also confusing.

It would be preferable to display some notion of the modifier eg "16 +3 = 19"

**Comment by frankieali** [comment]
Agreed. I'll test out that PR as soon as I'm able. It looks good at a glance. I'd like to see what happens when there are multiple modifiers or if the advanced roller is in play. E.g.: `3d6 + 4 + 4` or `3d6 + 2 + 1d8 + 4`. I don't recall if the advanced rollers grouping method works. I think anything related to the advanced roller should probably go into a separate issue though.
